odoo/addons/openupgrade_records/lib/apriori.py

```diff
@@ -10,6 +10,7 @@
 
 merged_modules = {
     # Odoo
+    'auth_brute_force': 'base',
```

Comment:
but `auth_brute_force` is not from `#Odoo`

please, put in a new `#OCA/server-auth` below like `OCA/web` case

Reply:
thanks @mreficent